GliaWidgets/Resources/Asset.swift

```diff
@@ -32,6 +32,7 @@ public enum Asset {
   public static let uploadRemove = ImageAsset(name: "uploadRemove")
   public static let chatPickMedia = ImageAsset(name: "chatPickMedia")
   public static let chatSend = ImageAsset(name: "chatSend")
+  public static let newMessageIndicator = ImageAsset(name: "newMessageIndicator")
```

Comment:
`unreadMessageIndicator`?

Reply:
I just used the name with which we always refer to that thingy. But this does sound better and has a closer meaning.